example.env

```diff
@@ -0,0 +1,3 @@
+DJANGO_SUPERUSER_USERNAME=admin
```

Comment:
This will not affect demo project. I'd rather add default env values to docker-compose environment section.